code/game/machinery/defibcharger.dm

```diff
@@ -87,25 +87,24 @@ obj/machinery/recharger/defibcharger/wallcharger/process()
 	return -1
 
 obj/machinery/recharger/defibcharger/wallcharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
-	if(..())
```

Comment:
As a child of machinery this has many more consequences to being removed than is apparent